ddtrace/debugging/_probe/model.py

```diff
@@ -247,6 +246,10 @@ class LogProbeMixin(AbstractProbeMixIn):
     take_snapshot: bool
     limits: CaptureLimits = field(compare=False)
 
+    @property
```

Comment:
I love this, as down the line we can provide the user to set the a probe limit on the UI